config/environments/development.rb

```diff
@@ -25,7 +25,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = true
+  config.assets.debug = ENV['ASSETS_DEBUG'] ? true : false
```

Comment:
Mostly I was trying to cut down on the noise in my terminal when I was trying to debug something. All the static assets being served were logging a success response and so it seemed less optimal to see them on every request.